cifer1/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "cifer1",
+  "assets": [
+    {
+      "description": "Decentralized Federated Learning",
+      "denom_units": [
+        {
+          "denom": "ucif",
+          "exponent": 0
+        },
+        {
+          "denom": "cif",
+          "exponent": 6
+        }
+      ],
+      "base": "ucif",
+      "name": "Cifer",
+      "display": "cif",
+      "symbol": "CIF",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cifer1/images/cif.png",
```

Comment:
I'm sorry, i didn't see that these references were updated. Can we please point them back to /cifer/images, and not /cifer1/images?